src/display/annotation_layer.js

```diff
@@ -2585,6 +2585,7 @@ class AnnotationLayer {
    */
   static render(parameters) {
     const { annotations, div, viewport, accessibilityManager } = parameters;
+    setLayerDimensions(div, viewport);
```

Comment:
While it might not technically matter, should we perhaps move the function-call in the same way in the `update`-method below to improve overall consistency?